cfssl.rb

```diff
@@ -8,6 +8,7 @@ class Cfssl < Formula
   head 'https://github.com/cloudflare/cfssl', :using => NoOpDownloadStrategy
   url "https://github.com/cloudflare/cfssl/archive/master.zip"
   homepage 'https://github.com/cloudflare/cfssl'
+  version '1.0.0'
```

Comment:
when is it packaged? The current version is 1.2 I think?

Reply:
This formula was pretty broken if you weren't installing from HEAD. I've updated it to build the binary and install it, as it's naturally not included in the source release.